starport_template/lib/pages/sign_transaction_page.dart

```diff
@@ -128,21 +128,23 @@ class _SignTransactionPageState extends State<SignTransactionPage> {
       builder: (context) => SizedBox(
         height: MediaQuery.of(context).size.height / 2.24,
         child: AssetsTransferSheet(
-          onTapDone: () async {
-            unawaited(
-              Navigator.of(context).pushAndRemoveUntil(
-                MaterialPageRoute(
-                  builder: (_) => const AssetsPortfolioPage(),
-                ),
-                (route) => false,
-              ),
-            );
+          onTapDone: () => _handleAssetTranserSheetDone(context),
+        ),
+      ),
+    );
+  }
 
-            await StarportApp.walletsStore.getBalances(selectedWallet.publicAddress);
-          },
+  Future<void> _handleAssetTranserSheetDone(BuildContext context) async {
```

Comment:
the rule for naming the user event methods is to call them `_onTap{what}` `_onLongPress{what}`, `onSwipe{{what}` and then within the method we decide what to do with the action. this way its a bit easier to comprehend the intention of the method and eventually modifying the logic inside of it does not require us to rename it

Reply:
Ahh I see I mostly default to using the handle `{what}` method no problem will modify and follow this, Is there a way we can document these project rules and all.